### DISCUSSION TITLE
Questions related to different languages

### DISCUSSION THREAD

**Discussion by mohithgupta** [Devorein/quizen#4] [Ideas]
Can we contribute by providing questions related to different languages? So that you can include them in the app?

**Comment by Devorein** [comment]
Oh my extremely sorry for the late response. I was not aware of this discussion. Could you please describe your issue a bit further? Do you want to create some more sample quizzes for different languages?

**Reply by mohithgupta** [reply]
Yes, I've prepared some questions for myself, and I think they will be a good add-on to the app. 
If we cannot contribute in such a way, Please let me know what things I can do to contribute to this project. 
Thank you.

**Reply by Devorein** [reply]
Sure if you can please send a PR, I'll take a look at the questions and merge them after fixing any issues. Also if you are familiar with react, you can contribute by integrating any other new features.
